libnopegl/src/backends/gl/glcontext.c

```diff
@@ -402,6 +407,43 @@ static int glcontext_probe_settings(struct glcontext *glcontext)
     return 0;
 }
 
+static int glcontext_check_driver(struct glcontext *glcontext)
+{
+    const char *gl_version = (const char *)ngli_glGetString(glcontext, GL_VERSION);
+    if (!gl_version) {
+        LOG(ERROR, "could not get OpenGL version");
+        return NGL_ERROR_BUG;
+    }
+
+    int mesa_version[3] = {0};
+    const char *mesa = strstr(gl_version, "Mesa");
+    if (mesa) {
+        int ret = sscanf(mesa,
+                         "Mesa %d.%d.%d",
+                         &mesa_version[0],
+                         &mesa_version[1],
+                         &mesa_version[2]);
+        if (ret != 3) {
+            LOG(ERROR, "could not parse Mesa version (%s)", mesa);
+            return NGL_ERROR_BUG;
+        }
+        LOG(INFO, "Mesa version: %d.%d.%d", NGLI_ARG_VEC3(mesa_version));
+    }
```

Comment:
You may want to add in the commit description why you add this unused code, that is I'm assuming because it's going to be used to disable the workaround when the fix is upstreamed. That being said, if the it's actually backported in all maintained releases, we may drop the workaround entirely.

Reply:
Done.